### ISSUE TITLE
Any PiP splitscreen - select/change any source

### ISSUE THREAD

**Issue by robotard** [webosbrew/apps-repo#87]
Please read conditions below carefully, and don't delete this part. Invalid issues will be closed.

* No apps related to piracy or intellectual property infringement is allowed.
* We can't guarantee if the app will be developed by anyone.
* We encourage you to develop the app by yourself. This would be a great contribution to webOS homebrew community.

- [ ] I understand conditions above
- [ ] (Optional) I'd like to develop this app by myself

*Issue content starts below this line*

---
A custom Picture in Picture (overlay, move, resize pip 2
Option to flip between 1&2 
Select any source/app for each pip panel….
(Screen one has audio by default)

Wayland composited windows (free as much memory as poss 🤷🏼‍♀️) ?
Perhaps run 2 apps at same time and trick them into sitting in our panes as if that was WebOS?

**Comment by mariotaku** [comment]
Hello! After getting a TV running webOS 8/9, I'm able to dig for a bit. This is possible with some caveats.

The list of allowed apps are defined by webOS. And they're modifiable with luna call `luna://com.webos.service.config/setConfigs` once you get root. Basically, you only need to add new app IDs to existing settings.

```json
{
  "configs": {
    "multiview.recommendedApps": [
      {
        "subApp": [
          {
            "appId": "com.webos.app.browser",
            "rank": 1
          },
          {
            "appId": "com.webos.app.camera",
            "rank": 2
          },
          {
            "appId": "com.webos.app.miracast",
            "rank": 3
          },
          {
            "appId": "youtube.leanback.v4",
            "rank": 4
          },
          {
            "appId": "spotify-beehive",
            "rank": 5
          },
          {
            "appId": "com.limelight.webos",
            "rank": 6
          },
          {
            "appId": "org.xbmc.kodi",
            "rank": 7
          }
        ],
        "mainAppId": "com.webos.app.livetv",
        "blacklistCountryGroup": [
          "CN",
          "HK"
        ]
      },
```

For some apps (like Moonlight), it seems that video can't work in multiview mode though.

I'll do researches and publish my findings when I have more resources, and hopefully we could make this feature more useful.

**Comment by kentyman23** [comment]
I notice Kodi is in the list. Did LG put it there? Does video work with it?

**Comment by mariotaku** [comment]
@kentyman23 No, I made the modification. You can add arbitrary apps to the list.
